namer/namer.cc

```diff
@@ -271,15 +269,13 @@ class NameInserter {
         return klass;
     }
 
-    bool handleNamerRewriter(core::MutableContext ctx, unique_ptr<ast::ClassDef> &klass,
+    void handleNamerRewriter(core::MutableContext ctx, unique_ptr<ast::ClassDef> &klass,
```

Comment:
While you're here, would you mind changing this back to `handleNamerDSL`? This looks like it was changed by a faulty sed.